### DISCUSSION TITLE
si5351 antenna pluging?

### DISCUSSION THREAD

**Discussion by whallmann** [mikaelnousiainen/RS41ng#56] [General]
Hi Mikael
saw the si5351 board connection on a photo in the slides. It looks like you did some transformation on the antenna with some resistors and condens.  Are there somewhere more details on this? maybe some more people here they played with transmittions ?
How many mW the board will transmitt?.   
best greetings
wolf DF7PN

**Comment by mikaelnousiainen** [comment]
@whallmann What you saw in the photo was a simple low-pass filter connected between the Si5351 output and a dipole antenna.

While the Si5351 transmissions generally work, there is a serious issue with the harmonics it generates, as the signal is not a pure sine wave -- far from it.

A simple filter might reduce the harmonics enough far from the sonde, but we noticed an issue with the Si5351 harmonics jamming the sonde GPS receiver, making it lose fix all the time. The Si5351 should probably be properly shielded and its output filtered better to have it not disturb GPS reception.

One has to remember that Si5351 is not a transmitter, but a clock generator.

The maximum transmit power (there are several choices) is in the order of 20 milliwatts IIRC.

**Reply by whallmann** [reply]
Thanks Mikael for the details. I guess this needs much Investigation and tests before start it to the air.  While on holidays i read some hours about LPFs. What kind of schematics did you used for this? 2nd or 3rd Order. something lightweight 😎😉
73 Wolf

**Reply by mikaelnousiainen** [reply]
I cannot remember exactly what kind of filter it was -- but just something simple with a couple of resistors and capacitors! Lightweight was the key.

**Reply by mikaelnousiainen** [reply]
Shielding is still the key -- the Si5351 will radiate on its own (even if there is a low-pass filter) and disturb the GPS.
